src/main/webapp/WEB-INF/themes/desktop/sass/sections/_header.scss

```diff
@@ -115,22 +115,8 @@ $width-nav: $block-grid-elements - $width-logo;
 #pagehdr .nav-main .logo {
   position: absolute;
   float: none;
-  margin-left: .625rem;
 }
 
-.ploscompbiol .logo {
```

Comment:
Why remove these heights?  We're only concerned about left padding in this ticket.

Reply:
Because I didn't understand why they were there & the images Sam supplied had different sizes for the different logos which I thought should govern the heights of the logos.

Reply:
Gotcha.  These styles were to reduce the width of the wider journal logos.  These new logos all have height 58px so I think these css styles are still needed.

Reply:
I should have added a comment to that effect

Reply:
thanks @sgatchell, fixed in 9e263c93a